content/migration.md

```diff
@@ -68,6 +68,170 @@ In the version 6 major release of NestJS, we introduced the code-first approach
 
 In order to migrate your existing application, simply rename all the `type-graphql` imports to the `@nestjs/graphql`.
 
+#### Terminus
+
+In the version 7 major release of `@nestjs/terminus`, a new simplified API has been introduced
+to run health checks. The previously required peer dependency [`@godaddy/terminus`](https://github.com/godaddy/terminus) has been removed, which allows us to integrate our health checks automatically into Swagger! Read more about the removal of `@godaddy/terminus` [here](https://github.com/nestjs/terminus/issues/340).
+
+##### Healthchecks
```

Comment:
Would you mind if we remove these h5 headings? For 2 reasons:
- consistency with other sections
- overall look

To migrate, everyone will have to go from top to bottom either way so grouping instructions into individual sections doesn't make too much sense.